services/sidecar/Dockerfile

```diff
@@ -61,7 +70,7 @@ WORKDIR /build
 COPY  --chown=scu:scu services/sidecar/requirements/*.txt \
       tmp/sidecar/requirements/
```

Comment:
Do you really need to create an intermediate folder ``/build/tmp`` ??  What is the rationale behind? is it to avoid override later? and therefore copy gets invalidated?
What about simply copying exactly _base.txt under ``/build``?

Reply:
well I did not check this as this was already in.. I'll have a look

Reply:
why do we need the build folder at all?

Reply:
cleanup done